src/encoded/static/components/experiment.js

```diff
@@ -27,622 +27,613 @@ const anisogenicValues = [
+                                    <div data-test="date-released">
+                                        <dt>Date released</dt>
+                                        <dd>{moment(context.date_released).format('MMMM D, YYYY')}</dd>
+                                    </div>
+                                : null}
+
+                                {seriesList.length ?
+                                    <div data-test="relatedseries">
+                                        <dt>Related datasets</dt>
+                                        <dd><RelatedSeriesList seriesList={seriesList} /></dd>
+                                    </div>
+                                : null}
+
+                                {context.submitter_comment ?
+                                    <div data-test="submittercomment">
+                                        <dt>Submitter comment</dt>
+                                        <dd>{context.submitter_comment}</dd>
+                                    </div>
+                                : null}
+
+                                <LibrarySubmitterComments replicates={replicates} />
+
+                                {context.internal_tags && context.internal_tags.length > 0 ?
+                                    <div className="tag-badges" data-test="tags">
+                                        <dt>Tags</dt>
+                                        <dd><InternalTags context={context} /></dd>
+                                    </div>
+                                : null}
+                            </dl>
                         </div>
-                    </PanelBody>
-                </Panel>
+                    </div>
+                </PanelBody>
+            </Panel>
 
-                {Object.keys(condensedReplicates).length ?
-                    <ReplicateTable condensedReplicates={condensedReplicates} replicationType={context.replication_type} />
-                : null}
+            {Object.keys(condensedReplicates).length > 0 ?
+                <ReplicateTable condensedReplicates={condensedReplicates} replicationType={context.replication_type} />
+            : null}
 
-                {/* Display the file widget with the facet, graph, and tables */}
-                <FileGallery context={context} encodevers={encodevers} anisogenic={anisogenic} />
+            {/* Display the file widget with the facet, graph, and tables */}
+            <FileGallery context={context} encodevers={encodevers} anisogenic={anisogenic} />
 
-                <FetchedItems {...this.props} url={experimentsUrl} Component={ControllingExperiments} />
+            <FetchedItems context={context} url={experimentsUrl} Component={ControllingExperiments} />
 
-                {combinedDocuments.length ?
-                    <DocumentsPanelReq documents={combinedDocuments} />
-                : null}
-            </div>
-        );
-    }
-}
+            {combinedDocuments.length ?
```

Comment:
All of these are because they're existing code from before we used AirBnB. It looks like new code, but it really is in the existing code. So should we change every single existing length comparison? I didn't think that was a good idea, but now I'm starting to lean that way.